bin/fixmystreet.com/hackney-noise

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env perl
+#
+# If a noise report within Hackney has not been updated
+# in four weeks and is open, close it with a message.
```

Comment:
This says 4 weeks but from what I can tell you need to supply the `--days` parameter to this script.

Do you think it should default to 28 days if there's no `--days` argument?